hwdb.d/60-keyboard.hwdb

```diff
@@ -1129,7 +1129,7 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO*:pn*Lenovo*V480*:pvr*
 # Lenovo ThinkCentre M800z/M820z/M920z AIO machines
 # key_scancode 00 is KEY_MICMUTE
 evdev:name:Microphone Mute Button:dmi:bvn*:bvr*:bd*:svnLENOVO*:pn*:*
- KEYBOARD_KEY_00=f20
```

Comment:
This one is weird, there is no driver in the kernel creating an input device with a name of "Microphone Mute Button" ...  Anyways lets just keep this updating the mapping to micmute.